src/compute/src/render/reduce.rs

```diff
@@ -1786,6 +1791,16 @@ mod monoids {
         Max(Row),
     }
 
+    impl ReductionMonoid {
+        /// Return the inner row independent of the enum variant.
+        fn row(&self) -> &Row {
+            use ReductionMonoid::*;
+            match self {
+                Min(row) | Max(row) => row,
+            }
+        }
+    }
```

Comment:
This seems to be pretty much the same implementation of `ReductionMonoid::finalize`? Perhaps we should just call this new method there to avoid code duplication.

Reply:
Merged!